test/e2e/framework.go

```diff
@@ -3069,6 +3083,10 @@ func getPingCommand(count int, size int, os string, ip *net.IP) []string {
 	if size != 0 {
 		cmd = append(cmd, sizeOption, strconv.Itoa(size))
 	}
+	if !fragment {
```

Comment:
Better to name it `disableFragment` and check as `if disableFragment {}`. By default, it will be false.

Reply:
Done.